integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -321,15 +354,21 @@ spec:
           {{- end }}
           {{- if .Values.mounts }}
             {{- range .Values.mounts }}
-            - name: "{{ .name }}"
-              mountPath: "{{ .path }}"
+          - name: "{{ .name }}"
+            mountPath: "{{ .path }}"
             {{- end }}
           {{- end }}
       restartPolicy: Always
       volumes:
         {{- if .Values.hub.enabled }}
         - name: conf-volume
           emptyDir: {}
+        - name: logs-volume
+          emptyDir: { }
```

Comment:
nit: `emptyDir: {}` (unless the correct style is actually `{ }`, idk)

Reply:
updated